LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/RegionCN470.cs

```diff
@@ -69,6 +71,14 @@ public RegionCN470()
             MaxADRDataRate = 7;
             RegionLimits = new RegionLimits((min: 470.3, max: 509.7), validDatarates, validDatarates, 0, 0);
 
+            UpstreamJoinFrequenciesToDownstreamAndChannelIndex = new Dictionary<double, (double, int)>
```

Comment:
Could we give names to the tuple elements, (just so we remember what they are in the future). Not sure the syntax is 100% thought
```suggestion
            UpstreamJoinFrequenciesToDownstreamAndChannelIndex = new Dictionary<double, (downstreamFreq: double, joinChannelIndex: int)>
```